espnet2/diar/decoder/linear_decoder.py

```diff
@@ -0,0 +1,28 @@
+# Licensed under the MIT license.
```

Comment:
Is there some reasons to use MIT license?

If you'll apply MIT license for diar task, you must put LICENSE file and also write clarify it in README.md.

Reply:
> Two high-level comments
> 
> * Please appropriately add some credits if you take it from EEND's sources and respect their work ;)
> * For this particular project, can you make a new file with the MIT license? (Just add a `# Licensed under the MIT license.` in the header) if it is not complicated?

I was following @sw005320 's comment. This task is developed based on https://github.com/hitachi-speech/EEND , where they used the MIT license. I will clarify that later.

Reply:
@sw005320 @ftshijt 
MIT license doesn't forbid to change the LICENSE from the original. Regardless of changing or keeping the original LICENSE, only you have to do is clarifying the original copyright and pointing the URL of MIT LICENSE or writing the full text of the LICENSE. We should use same LICENSE as espnet, i.e. apache, if there are not your personal reason to use MIT LICENSE.